lib/dom.js

```diff
@@ -937,7 +931,7 @@ declare class HTMLSelectElement extends HTMLElement {
   value: string;
 
   add(element: HTMLElement, before?: HTMLElement): void;
-  remove(index: number): void;
+  //remove(index: number): void;
```

Comment:
add this back but make index optional (`index?: number`), per https://html.spec.whatwg.org/multipage/forms.html#dom-select-remove

what a crazy API. >:(